addons/survey/static/src/scss/survey_form.scss

```diff
@@ -0,0 +1,251 @@
+.o_survey_header {
```

Comment:
Is it absolutely necessary to change the style linked to the generic frontend layout ? I feel like it should not be done.